src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef(__WINDOWS__)
+#include "resource.h"
+#else
 #include "icon.png.xxd"
+#endif
```

Comment:
K, done. It may only be 1kb now, but someone may build their own with a different icon of higher quality/size, so it could blow up a bit. Better safe than sorry.